### DISCUSSION TITLE
Display Field's Data Type on System Collections

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#14223] [Feature Requests]
**I feel like it would be nice if we had data types listed on system collections. Is there a specific reason we don't do this?**

https://user-images.githubusercontent.com/62306630/176733829-79340e12-5e3d-4c66-a83e-4364c5e88b49.mov

**Comment by rijkvanzanten** [comment]
No real reason. I also noticed the labels there aren't in monospace for some reason, so worth looking into that as well 👍🏻

**Comment by azrikahar** [comment]
Implemented in 9.14.1 via #14321! 👍
